From c31690dd43f33e4710043247247fc996c0a57994 Mon Sep 17 00:00:00 2001
From: Sebastian Parschauer <s.parschauer@gmx.de>
Date: Fri, 6 Jan 2017 08:57:37 +0100
Subject: scanmem.1: Escape leading dot in scanmem output

On Debian, Lintian reports a man warning for scanmem.1:

$ man --warnings ./scanmem.1 >/dev/null
<standard input>:387: warning: macro `.........info:' not defined

The man command tries to parse this as a macro which is unknown of
cause. So replace the first dot of that line with "\[char46]".

Reference:
http://stackoverflow.com/questions/11469341/escaping-dots-in-groff

Last-Update: 2017-01-06

Patch-Name: scanmem-man-escape-leading-dot.patch
---
 scanmem.1 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scanmem.1 b/scanmem.1
index d87bad7..f697f60 100644
--- a/scanmem.1
+++ b/scanmem.1
@@ -384,7 +384,7 @@ the new value, 83.
 
 .nf
 16> 83
-..........info: we currently have 1 matches.
+\[char46].........info: we currently have 1 matches.
 info: match identified, use "set" to modify value.
 info: enter "help" for other commands.
 1> list
